# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../raptor/raptor-1.4.21-remove-deprecated-headers.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

The use of including <curl/types.h> was deprecated more than 7 years ago,
and the header file was finally removed with the release of curl 7.21.7.

Reference how upstream changed their own example code:
https://github.com/bagder/curl/commit/ac28971aa61d28e5dd54888e34e958d1c742b461

--- raptor-1.4.21/src/raptor_internal.h.orig	2012-05-09 10:29:36.817043899 +0200
+++ raptor-1.4.21/src/raptor_internal.h	2012-05-09 10:30:16.905794279 +0200
@@ -852,8 +852,6 @@
 
 #ifdef RAPTOR_WWW_LIBCURL
 #include <curl/curl.h>
-#include <curl/types.h>
-#include <curl/easy.h>
 #endif
 
 /* Size of buffer used in various raptor_www places for I/O  */
